eo-parser/src/main/resources/org/eolang/parser/move-versions-down.xsl

```diff
@@ -0,0 +1,56 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+The MIT License (MIT)
+
+Copyright (c) 2016-2023 Objectionary.com
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included
+in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+SOFTWARE.
+-->
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" id="move-versions-down" version="2.0">
+  <!--
+  Here we add empty "ver" attribute for all objects, which "base" starts with "org.eolang."
```

Comment:
@maxonfjvipon Is `move-versions-down.xsl` a suitable name for transformation with that description:
> Here we add empty "ver" attribute for all objects, which "base" starts with "org.eolang.

Why "down"?

Reply:
@volodya-lombrozo the description is wrong, I will fix it. The version should move inwards to "org" object.
From this:
```xml
<o ver="123">
  <o>
    <o>
      <o base="org">
        <o base="Q"/>
      </o>
    </o>
  </o>
</o>
```
To this:
```xml
<o>
  <o>
    <o>
      <o base="org" ver="123">
        <o base="Q"/>
      </o>
    </o>
  </o>
</o>
```